cms/djangoapps/contentstore/views/course.py

```diff
@@ -1158,6 +1158,11 @@ def settings_handler(request, course_key_string):
                 if is_prerequisite_courses_enabled():
                     prerequisite_course_keys = request.json.get('pre_requisite_courses', [])
                     if prerequisite_course_keys:
+                        run_extension_point(
+                            'PEARSON_CORE_MILESTONE_PREREQUISITES_MODULE',
+                            course_key = course_key,
+                            prerequisite_course_keys = prerequisite_course_keys,
+                        )
                         if not all(is_valid_course_key(course_key) for course_key in prerequisite_course_keys):
                             return JsonResponseBadRequest({"error": _("Invalid prerequisite course key")})
                         set_prerequisite_courses(course_key, prerequisite_course_keys)
```

Comment:
What do you think about moving the `run_extension_point` part after the validation? @anfbermudezme 
```suggestion
                        if not all(is_valid_course_key(course_key) for course_key in prerequisite_course_keys):
                            return JsonResponseBadRequest({"error": _("Invalid prerequisite course key")})
                        run_extension_point(
                            'PEARSON_CORE_MILESTONE_PREREQUISITES_MODULE',
                            course_key = course_key,
                            prerequisite_course_keys = prerequisite_course_keys,
                        )
                        set_prerequisite_courses(course_key, prerequisite_course_keys)
```

Reply:
What happens if `run_extension_point(..` goes after `set_prerequisite_courses(..`?. Does the order have any impact in the behavior? @anfbermudezme